api/chat.py

```diff
@@ -1,246 +0,0 @@
-# ------------------------------- env, constants -------------------------------
-from followups import multisearch_authored
-from get_blocks import get_top_k_blocks, Block
```

Comment:
these lines were changed after this was moved, so git thinks this was deleted, rather than moved